chapter_1/Exeption handling/4°_tryal.py

```diff
@@ -0,0 +1,7 @@
+import socket
+socket.setdefaulttimeout(2)
+s = socket.socket()
```

Comment:
Please add a newline (blank line) here to keep things easy to consume.

Reply:
Done.